src/utils/viewer/viewer.cpp

```diff
@@ -318,6 +319,15 @@ Key Commands:
   std::unique_ptr<ObjectPickingHelper> objectPickingHelper_;
   // returns the number of visible drawables (meshVisualizer drawables are not
   // included)
+
+  //Profiling
+   Mn::DebugTools::GLFrameProfiler _profiler{
```

Comment:
```suggestion
   Mn::DebugTools::GLFrameProfiler profiler_{
```

We use postfix `_` to denote member variables